drivers/hmis/app/graphql/types/hmis_schema/custom_case_note.rb

```diff
@@ -17,6 +17,7 @@ class HmisSchema::CustomCaseNote < Types::BaseObject
     field :client, HmisSchema::Client, null: false
     field :content, String, null: false
     field :information_date, GraphQL::Types::ISO8601Date, null: true
+    field :form_definition_id, ID, null: false
```

Comment:
I think we will want to add this field to all types of records that have Form Processors tied to them, to support https://github.com/open-path/Green-River/issues/6113. lets discuss that issue again next week